_locales/en/messages.json

```diff
@@ -683,7 +683,7 @@
         "message": "Disarm motors after set delay [seconds] (Requires MOTOR_STOP feature)"
     },
     "configurationDisarmKillSwitch": {
-        "message": "Disarm motors regardless of throttle value (When arming via AUX channel)"
+        "message": "Arm/Disarm motors regardless of throttle value when using AUX channel"
```

Comment:
This is clearly wrong. This switch only affects how **dis**arming works, throttle has to be low for arming even with this switch on. The way you worded the message, it sounds like with the switch on, arming will be possible at any throttle value, which is wrong (and would be **very** dangerous if it weren't).

Like i said in https://github.com/betaflight/betaflight-configurator/issues/597#issuecomment-322943541, my suggestion is `Disarm motors regardless of throttle value (when arming / disarming is configured to be via AUX channel)`.